src/requirements/data/colleges/en.ts

```diff
@@ -105,61 +106,73 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
+  //       engineeringLiberalArtsDistributions.some(
+  //         distribution => course.catalogDistr?.includes(distribution) ?? false
+  //       ),
+  //   ],
+  //   fulfilledBy: 'credits',
+  //   perSlotMinCount: [18],
+  //   allowCourseDoubleCounting: true,
+  // },
+  // TODO: Create special function for this as it is the same as Advisor-Approved Electives for CS checker
   {
-    name: 'Liberal Studies Distribution: 3 categories',
+    name: 'Advisor-Approved Electives',
     description:
-      'In addition to six courses, the liberal studies courses must be from 3 categories.',
+      'At least 3 credit hours total. All academic courses count.' +
+      'No PE courses, courses numbered 10xx, and ROTC courses below the 3000-level allowed.',
     source:
-      'https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies',
+      'https://www.engineering.cornell.edu/students/undergraduate-students/curriculum/undergraduate-requirement',
     checker: [
-      (course: Course): boolean =>
-        (course.catalogDistr?.includes('LA') || course.catalogDistr?.includes('LAD')) ?? false,
-      ...engineeringLiberalArtsDistributions
-        .filter(it => it !== 'LA' && it !== 'LAD')
-        .map(distribution => (course: Course): boolean =>
-          course.catalogDistr?.includes(distribution) ?? false
-        ),
-    ],
-    fulfilledBy: 'courses',
-    perSlotMinCount: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
-    minNumberOfSlots: 3,
-    allowCourseDoubleCounting: true,
-  },
-  {
-    name: 'Liberal Studies Distribution: 18 credits',
-    description:
-      'In addition to six courses, the liberal studies distribution must total a minimum of 18 credits.',
-    source:
-      'https://www.engineering.cornell.edu/students/undergraduate-students/advising/liberal-studies',
-    checker: [
-      (course: Course): boolean =>
-        engineeringLiberalArtsDistributions.some(
-          distribution => course.catalogDistr?.includes(distribution) ?? false
-        ),
+      (course: Course): boolean => {
+        const { catalogNbr } = course;
+        return !(ifCodeMatch(course.subject, 'PE') && ifCodeMatch(catalogNbr, '10**'));
+      },
     ],
+    checkerWarning: 'We do not check that the courses are major approved.',
     fulfilledBy: 'credits',
-    perSlotMinCount: [18],
-    allowCourseDoubleCounting: true,
-  },
-  {
-    name: 'Advisor-Approved Electives',
-    description:
-      'Six credits of electives are required and must be approved by the student’s faculty advisor.',
-    source:
-      'https://www.engineering.cornell.edu/students/undergraduate-students/curriculum/undergraduate-requirements',
-    fulfilledBy: 'self-check',
-    minCount: 6,
+    perSlotMinCount: [3],
   },
-  // TODO: INFO 1200 currently fulfills Eng Comm for ISST majors but should not be double-counted
+  // TODO: Remove warning once we can filter reqs based on majors (ISST) and petitions
```

Comment:
By default, INFO 1200 automatically fulfills this and can't be changed for the ISST major. This is wrong. Because we can't support petitions and don't have the option to edit requirements either, I changed this to include warnings so the req isn't wrong for ISST majors and more open ended